frontend/amundsen_application/api/metadata/v0.py

```diff
@@ -776,6 +776,8 @@ def _get_dashboard_metadata(*, uri: str, index: int, source: str) -> None:
         url = f'{app.config["METADATASERVICE_BASE"]}{DASHBOARD_ENDPOINT}/{uri}'
 
         response = request_metadata(url=url, method=request.method)
+        if response.status_code != 200:
```

Comment:
can you use `HTTPStatus.OK` instead of `200`?

Reply:
If you want to run with these changes go for it. But I made this PR  2 years ago and I can't really shift to working back on this project right now.